airbyte-integrations/bases/base-normalization/integration_tests/dbt_integration_test.py

```diff
@@ -415,7 +415,7 @@ def writer():
                             line = input_data.readline()
                             if not line:
                                 break
-                            if not line.startswith(b"#"):
+                            if not line.startswith(b"//"):
```

Comment:
should both comment types be considered?

Reply:
this is only applied to the `messages*.txt` files. I'd prefer for us to just pick a convention and use it everywhere (and my vs code extension is only highlighting `//` :P )